lib/config.js

```diff
@@ -436,8 +436,38 @@ var parseConfig = function (configFilePath, cliOptions) {
   return normalizeConfig(config, configFilePath)
 }
 
+var directConfig = function (configCallback) {
+  if (typeof configCallback === 'function') {
+    log.debug('Loading config from provided callback')
```

Comment:
This debug msg makes it sound like action is following immediately.  Maybe "Config callback detected".

Reply:
Agreed. Change incoming.